packages/lib/modules/pool/PoolList/PoolListFilters.tsx

```diff
@@ -643,19 +643,21 @@ export function PoolListFilters() {
             </PopoverContent>
           </Box>
         </Popover>
-        <Button
-          as={Link}
-          display="flex"
-          gap="2"
-          href={poolCreatorUrl}
-          isExternal
-          ml="ms"
-          onClick={() => trackEvent(AnalyticsEvent.ClickPoolListCreatePool)}
-          variant="tertiary"
-        >
-          <Icon as={Plus} boxSize={4} />
-          {!isMobile && 'Create pool'}
-        </Button>
+        {!isBalancer && (
+          <Button
+            as={Link}
+            display="flex"
+            gap="2"
+            href={poolCreatorUrl}
+            isExternal
+            ml="ms"
+            onClick={() => trackEvent(AnalyticsEvent.ClickPoolListCreatePool)}
+            variant="tertiary"
+          >
+            <Icon as={Plus} boxSize={4} />
+            {!isMobile && 'Create pool'}
+          </Button>
+        )}
```

Comment:
<img width="696" height="111" alt="Image" src="https://github.com/user-attachments/assets/cb0713fe-e8d2-4f76-ae9a-1f7e0968ad42" />

For Beets, :point_up:  is the only way to get to creator because 

no "Build" popover 

<img width="743" height="128" alt="Image" src="https://github.com/user-attachments/assets/dd0792ee-5c6b-45ed-86e6-d19ca7212659" />


no bottom of page promo 

<img width="1314" height="252" alt="Image" src="https://github.com/user-attachments/assets/d32452ff-5f81-444d-86b5-420e4e24bff6" />